src/main/java/cart/domain/Member.java

```diff
@@ -0,0 +1,67 @@
+package cart.domain;
+
+import static cart.exception.ErrorCode.MEMBER_EMAIL_LENGTH;
+import static cart.exception.ErrorCode.MEMBER_NICKNAME_LENGTH;
+
+import cart.exception.GlobalException;
+
+public class Member {
+
+    private static final int EMAIl_MIN_LENGTH = 5, EMAIL_MAX_LENGTH = 50;
+    private static final int NICKNAME_MIN_LENGTH = 1, NICKNAME_MAX_LENGTH = 30;
+
+    private final String email;
+    private final MemberPassword password;
+    private final String nickname;
+    private final String telephone;
+
+    private final MemberRole role;
+
+    public Member(final String email, final MemberPassword password, final String nickname,
```

Comment:
```suggestion
    private Member(final String email, final MemberPassword password, final String nickname,
```

팩토리 메서드가 있으니 생성자는 private으로 닫을 수 있겠군요 ㅎㅎ 다른 곳은 잘 해주셨네요